common/v2/routing/routes.tsx

```diff
@@ -1,37 +1,144 @@
-import React from 'react';
+import React, { lazy } from 'react';
 import { Redirect } from 'react-router-dom';
 
-import { IAppRoute } from '@types';
-import { ROUTE_PATHS, IS_ACTIVE_FEATURE } from '@config';
-import {
-  AddAccountFlow,
-  CreateWallet,
-  Mnemonic,
-  Keystore,
-  Dashboard,
-  Settings,
-  Import,
-  Export,
-  DownloadApp,
-  NoAccounts,
-  ScreenLockNew,
-  ScreenLockLocked,
-  ScreenLockForgotPassword,
-  SendAssets,
-  ReceiveAssets,
-  SwapAssetsFlow,
-  Home,
-  SignAndVerifyMessage,
-  BroadcastTransactionFlow,
-  InteractWithContractsFlow,
-  DeployContractsFlow,
-  DeFiZapFlow,
-  PurchaseMembershipStepper,
-  MembershipEducation,
-  BuyAssets
-} from '@features';
+import { IAppRoute } from 'v2/types';
+import { ROUTE_PATHS, IS_ACTIVE_FEATURE } from 'v2/config';
+import { Home, Dashboard, NoAccounts } from 'v2/features';
 import { requiresDesktopApp } from './helpers';
 
+const CreateWallet = lazy(() =>
+  import(
+    /* webpackChunkName: "CreateWallet" */
+    'v2/features/CreateWallet/CreateWallet'
+  )
+);
```

Comment:
1. Can we use aliases eg. `@features` and index files for the imports?
2. Can we create these dynamically? Something like:
```
const COMPONENTS = {
 AddAccountFlow: '@features/AddAccount'
}

const lazify = (v, k) => lazy(() => import(`${v}))
const lazyComponents = mapObjIndexed(lazify, COMPONENTS)
```
3. You mentioned that during experimentation the bundle size would increase. Could it be because of each of these includes have a lot of shared components between them that are included in each bundle? Would it be better to create larger groups like (AddAccount + SendAssets + SwapAssets)?

Reply:
1. Yeah, this was already rebased;
2. Not really, because it needs to be statically available. Also just tried it.
3. Yeah, this is my next step, decreasing main bundle size, and joining components together, so that each separate page bundle gets bigger. But this would also mean, that reexporting some components would not be a wise thing to do, since webpack, can't decide, if they can be joined in separate page bundle and main bundle.